lib/services/doctor-service.ts

```diff
@@ -26,96 +26,118 @@ class DoctorService implements IDoctorService {
-				this.$logger.out("You will not be able to build your projects for iOS if they contain plugin with CocoaPod file." + EOL
-					+ "To be able to build such projects, verify that you have installed CocoaPods.");
+			if (!sysInfo.androidInstalled) {
+				this.$logger.warn("WARNING: The Android SDK is not installed or is not configured properly.");
+				this.$logger.out("You will not be able to build your projects for Android and run them in the native emulator." + EOL
+					+ "To be able to build for Android and run apps in the native emulator, verify that you have" + EOL
+					+ "installed the latest Android SDK and its dependencies as described in http://developer.android.com/sdk/index.html#Requirements" + EOL
+				);
+
+				this.printPackageManagerTip();
 				result = true;
 			}
 
-			if (sysInfo.xcodeVer && sysInfo.cocoapodVer) {
-				let problemWithCocoaPods = this.verifyCocoaPods();
-				if (problemWithCocoaPods) {
-					this.$logger.warn("WARNING: There was a problem with CocoaPods");
-					this.$logger.out("Verify that CocoaPods are configured properly.");
+			if (this.$hostInfo.isDarwin) {
+				if (!sysInfo.xcodeVer) {
+					this.$logger.warn("WARNING: Xcode is not installed or is not configured properly.");
+					this.$logger.out("You will not be able to build your projects for iOS or run them in the iOS Simulator." + EOL
+						+ "To be able to build for iOS and run apps in the native emulator, verify that you have installed Xcode." + EOL);
+					result = true;
+				}
+
+				if (!sysInfo.cocoapodVer) {
+					this.$logger.warn("WARNING: CocoaPods is not installed or is not configured properly.");
+					this.$logger.out("You will not be able to build your projects for iOS if they contain plugin with CocoaPod file." + EOL
+						+ "To be able to build such projects, verify that you have installed CocoaPods.");
+					result = true;
+				}
+
+				if (sysInfo.xcodeVer && sysInfo.cocoapodVer) {
+					let problemWithCocoaPods = this.verifyCocoaPods();
+					if (problemWithCocoaPods) {
+						this.$logger.warn("WARNING: There was a problem with CocoaPods");
+						this.$logger.out("Verify that CocoaPods are configured properly.");
+						result = true;
+					}
+				}
+
+				if (sysInfo.cocoapodVer && semver.valid(sysInfo.cocoapodVer) === null) {
+					this.$logger.warn(`WARNING: CocoaPods version is not a valid semver version.`);
+					this.$logger.out("You will not be able to build your projects for iOS if they contain plugin with CocoaPod file." + EOL
+						+ `To be able to build such projects, verify that you have at least ${DoctorService.MIN_SUPPORTED_POD_VERSION} version installed.`);
 					result = true;
 				}
+
+				if (sysInfo.cocoapodVer && semver.valid(sysInfo.cocoapodVer) && semver.lt(sysInfo.cocoapodVer, DoctorService.MIN_SUPPORTED_POD_VERSION)) {
+					this.$logger.warn(`WARNING: CocoaPods version is lower than ${DoctorService.MIN_SUPPORTED_POD_VERSION}.`);
```

Comment:
WARNING: Your current CocoaPods version is lower than ${DoctorService.MIN_SUPPORTED_POD_VERSION}.`